### ISSUE TITLE
Fix random tab jumps

### ISSUE THREAD

**Issue by siemiatj** [metasfresh/metasfresh-webui-frontend#2246]
### Is this a bug or feature request?

Bug

### What is the current behavior?

> i noticed sometimes when using mouse to select sth, using Tab afterwards lets you jump not  in the following field, but somewhere else completely. e.g. selecting a price system in bpartner window, customer tab, with mouse: when using Tab afterwards, the notifications open; when you use Enter to select the price system, the following field gets selected after Tab:

http://recordit.co/20CGJRLdoa

#### Which are the steps to reproduce?

> happening when creating a new entry in window/540120 or window/540113: using Tab opens the notifications after selecting with mouse

### What is the expected or desired behavior?

Tab should behave in an expected way.

This is a follow up to #2244

**Comment by ionut-movila** [comment]
I had a debugging session here with @oliviupis and what is happening in page http://localhost:3000/window/123/2156491, 3rd tab at the bottom (Define Customer Accounting) is like this:

1. Select a table cell to edit content, by double click it.
2. Click one more time to open dropdown content.
3. Select a value from the dropdown (new value or the same does not matter)

As a result: table cell looses focus and when pressing Tab it will open the Inbox/Notifications menu (that has tabindex=0).

@siemiatj  We will have to discuss the logic here for handling tab logic and shortcuts. Thank you.

**Comment by siemiatj** [comment]
Yeah, I guess we can skip this for now then. As mentioned before I've pretty much rewritten the table navigation from scratch here https://github.com/metasfresh/metasfresh-webui-frontend/pull/2250 so it makes no sense to fix it now, and then (probably) do it again later.

**Comment by ionut-movila** [comment]
With the updated from PR: #2276 at least we have fixed "notifications menu open". Now what happens is that after you edit a field you need to press twice TAB to advance to next cell.
